src/lib/server/models/discord/attachment/index.ts

```diff
@@ -0,0 +1,17 @@
+import { type InferOutput, nullable, number, object, optional, string } from 'valibot';
+import { Snowflake } from '$lib/server/models/discord/snowflake';
+
+export const Attachment = object({
+    id: optional(Snowflake),
+    filename: string(),
+    title: optional(string()),
+    description: optional(string()),
+    content_type: nullable(optional(string())),
+    size: optional(number()),
+    url: string(),
+    proxy_url: optional(string()),
+    height: optional(number()),
+    width: optional(number()),
```

Comment:
Surprisingly, these are also nullable according to the [docs](https://discord.com/developers/docs/resources/message#attachment-object)!
```suggestion
    height: optional(nullable(number())),
    width: optional(nullable(number())),
```